embrace-android-core/src/test/java/io/embrace/android/embracesdk/internal/ndk/symbols/SymbolServiceImplTest.kt

```diff
@@ -1,41 +1,36 @@
 package io.embrace.android.embracesdk.internal.ndk.symbols
 
-import android.content.Context
-import android.content.res.Resources
 import android.util.Base64
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import io.embrace.android.embracesdk.fakes.FakeDeviceArchitecture
 import io.embrace.android.embracesdk.fakes.FakeEmbLogger
 import io.embrace.android.embracesdk.fakes.TestPlatformSerializer
+import io.embrace.android.embracesdk.fakes.config.FakeBase64SharedObjectFilesMap
+import io.embrace.android.embracesdk.fakes.config.FakeInstrumentedConfig
 import io.embrace.android.embracesdk.internal.payload.NativeSymbols
-import io.mockk.every
-import io.mockk.mockk
 import org.junit.Assert.assertEquals
 import org.junit.Assert.assertNull
 import org.junit.Test
 import org.junit.runner.RunWith
 
+// Needed for android.util.Base64
```

Comment:
Could we tweak it to Kotlin/Java Base64 so the test can run on the JVM? Can be ignored if that's non-trivial

Reply:
We can't. We are using the Android Base64 decoder in the SDK, and we can't swap it with the Java one because it's only compatible with API > 26

Reply:
We could use the Kotlin one though, if we are okay with the experimental annotation.

Reply:
(we can actually use the java.util.Base64 class for encoding in the test, but we will still need the JUnit runner so the decoding  works)

Reply:
Let's keep this as-is for now